<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendsView</class>
 <widget class="QDialog" name="LegendsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>611</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="EditorlLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gamepageicon.qrc">
          <normaloff>:/gpicon/images/icon/ListAdd.png</normaloff>:/gpicon/images/icon/ListAdd.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gamepageicon.qrc">
          <normaloff>:/gpicon/images/icon/ListChange.png</normaloff>:/gpicon/images/icon/ListChange.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gamepageicon.qrc">
          <normaloff>:/gpicon/images/icon/ListDelete.png</normaloff>:/gpicon/images/icon/ListDelete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gamepageicon.qrc">
          <normaloff>:/gpicon/images/icon/ListSearch.png</normaloff>:/gpicon/images/icon/ListSearch.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="selectLayout">
      <item>
       <widget class="QLineEdit" name="infoLine">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="timerProgressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="LegendsTableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>30</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: rgb(66, 133, 244); 
border-radius: 8px; 
color:rgb(255, 255, 255); 
} 

QPushButton:hover 
{ 
background-color:rgb(42,86,198); 

}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>30</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: rgb(66, 133, 244); 
border-radius: 8px; 
color:rgb(255, 255, 255); 
} 

QPushButton:hover 
{ 
background-color:rgb(42,86,198); 

}
</string>
   </property>
   <property name="text">
    <string>确  定</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="gamepageicon.qrc"/>
 </resources>
 <connections/>
</ui>
